### PR TITLE
Installment source type

### PR DESCRIPTION
**Pull request by Namiii**
In this PR, we've added the support for installments through adding new source types. The 5 new types are as follows: 

1. "installment_bay"
2. "installment_first_choice"
3. "installment_bbl"
4. "installment_ktc"
5. "installment_kbank"

We've also added `installment_terms` to both `CreateRequestBuilder` and `Source` to handle creating an installment request and its response.

### CODE REVIEW

File: src/main/java/co/omise/Example.java [hunk 1]
```diff
@@ -419,8 +419,20 @@ void createSource() throws IOException, OmiseException, ClientException {
 
         Source source = client().sendRequest(request);
         System.out.printf("source created: %s", source.getId());
-}
-  
+    }
+
+    void createSource_installment() throws IOException, ClientException, OmiseException {
```
Comment (nuxzero) [inline src/main/java/co/omise/Example.java]: Not sure about mixing `_` in the method name. Could you change to camel-case?

File: src/main/java/co/omise/models/Source.java [hunk 2]
```diff
@@ -121,6 +131,8 @@ public void setTerminalId(String terminalId) {
         private String storeName;
         @JsonProperty("terminal_id")
         private String terminalId;
+        @JsonProperty("installment_terms")
+        private String installmentTerms;
```
Comment (nuxzero) [inline src/main/java/co/omise/models/Source.java]: Is it should be the integer?

File: src/test/java/co/omise/requests/SourceRequestTest.java [hunk 3]
```diff
@@ -25,4 +26,23 @@ public void testCreate() throws IOException, OmiseException {
         assertEquals("redirect", source.getFlow().toString());
         assertEquals("alipay", source.getType().toString());
     }
+
+    @Test
+    public void testCreate_installment() throws IOException, OmiseException {
```
Comment (nuxzero) [inline src/test/java/co/omise/requests/SourceRequestTest.java]: Camel-case?

File: src/main/java/co/omise/Example.java (hunk 1 above)
Comment (Namiii) [reply src/main/java/co/omise/Example.java]: Sure

File: src/main/java/co/omise/models/Source.java (hunk 2 above)
Comment (Namiii) [reply src/main/java/co/omise/models/Source.java]: The return type was a String, so I set it as String everywhere to keep it consistent.

File: src/test/java/co/omise/requests/SourceRequestTest.java (hunk 3 above)
Comment (Namiii) [reply src/test/java/co/omise/requests/SourceRequestTest.java]: But we do add `_` in the between for test methods right?

File: src/main/java/co/omise/models/Source.java (hunk 2 above)
Comment (nuxzero) [reply src/main/java/co/omise/models/Source.java]: 👌

**Comment by Namiii** [comment]
@nuxzero All changes done 👍

**Pull request merged**